src/legacy/ui/public/field_editor/field_editor.js

```diff
@@ -331,6 +338,51 @@ export class FieldEditorComponent extends PureComponent {
     );
   }
 
+  /**
+   * renders a warning and a table of conflicting indices
+   * in case there a indices with different types
+   */
+  renderTypeConflict(conflictDescriptions) {
+    const { intl } = this.props;
+
+    const columns = [{
+      field: 'type',
+      name: intl.formatMessage({ id: 'common.ui.fieldEditor.typeLabel', defaultMessage: 'Type' }),
+      width: '100px',
+    }, {
+      field: 'indices',
+      name: intl.formatMessage({ id: 'common.ui.fieldEditor.indexNameLabel', defaultMessage: 'Index Names' })
+    }];
+
+    const items = Object
+      .entries(conflictDescriptions)
+      .map(([type, indices]) => ({
+        type,
+        indices: Array.isArray(indices) ? indices.join(', ') : ''
+      }));
+
+    return (
+      <span>
+        <EuiCallOut
+          color="warning"
+          iconType="alert"
+          title={<FormattedMessage id="common.ui.fieldEditor.fieldTypeConflict" defaultMessage="Field Type Conflict"/>}
```

Comment:
Use sentence case for all titles:

```suggestion
          title={<FormattedMessage id="common.ui.fieldEditor.fieldTypeConflict" defaultMessage="Field type conflict"/>}
```